--- a/configure.ac
+++ b/configure.ac
@@ -10,6 +10,7 @@
 
 AC_CONFIG_SRCDIR([src/gstdvswitchsrc.c])
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_MACRO_DIR([m4])
 
 dnl required version of automake
 AM_INIT_AUTOMAKE([1.10])
@@ -45,7 +46,6 @@
 PKG_CHECK_MODULES(GST, [
   gstreamer-0.10 >= $GST_REQUIRED
   gstreamer-base-0.10 >= $GST_REQUIRED
-  gstreamer-controller-0.10 >= $GST_REQUIRED
 ], [
   AC_SUBST(GST_CFLAGS)
   AC_SUBST(GST_LIBS)
